build.sh

```diff
@@ -231,7 +234,7 @@ command_test() {
     echo "::group::Security tests"
     echo "## Testing for outdated packages and security issues... ##"
     execute $python_binary -m pip list --outdated --format=columns
-    execute $python_binary -m pip install $PIP_ARGS safety
+    execute $python_binary -m pip install $PIP_ARGS safety=="$SAFETY_VERSION"
```

Comment:
I see that `safety` fetches its database from the web, so pinning its version is not a problem (and should fail noisily if incompatibilities arise).